File: src/examples/php/rates/retrieve-rate.php
<?php

require_once '/Users/jhammond/git/easypost/easypost-tools/vendor/autoload.php';

use EasyPost\EasyPost;
use EasyPost\Error;
use EasyPost\Rate;
use Dotenv\Dotenv;

$dotenv = Dotenv::createImmutable('/Users/jhammond/git/easypost/easypost-tools');
$dotenv->load();
EasyPost::setApiKey($_ENV['EASYPOST_TEST_API_KEY']);

try {
    $rate = Rate::retrieve('rate_123...');
    echo $rate;
} catch (Error $exception) {
    echo $exception;
}

File: src/examples/ruby/rates/retrieve_rate.rb
# frozen_string_literal: true

require 'easypost'
require 'dotenv'
require 'amazing_print'

Dotenv.load('/Users/jhammond/git/easypost/easypost-tools/.env')
EasyPost.api_key = ENV['EASYPOST_TEST_API_KEY']

rate = EasyPost::Rate.retrieve('rate_123...')

ap rate
